.travis.yml

```diff
@@ -4,6 +4,7 @@ addons:
 before_install:
  - git branch -vv | grep '^*'
  - pwd
+ - find . -name "*.sh" -exec bash -n {} \;
```

Comment:
Kind of syntax check/lint method with shellscript:
https://stackoverflow.com/questions/171924/how-do-i-check-syntax-in-bash-without-running-the-script